packages/reapit-connect-management/public/index.html

```diff
@@ -4,7 +4,7 @@
     <base href="/" />
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
-    <title>Foundations App</title>
+    <title>Reapit Management</title>
```

Comment:
Should be Reapit Connect Management

Reply:
- `Rename tab name from 'Foundations App' to ‘Reapit Management’` in ticket's description cc @HollyJoyPhillips